apps/bfd-db-migrator/src/main/java/gov/cms/bfd/migrator/app/MigratorApp.java

```diff
@@ -19,8 +41,92 @@
    */
   public static void main(String[] args) {
     LOGGER.info("Successfully started");
-    LOGGER.info("Hello, World!");
+
+    AppConfiguration appConfig = null;
+    try {
+      appConfig = AppConfiguration.readConfigFromEnvironmentVariables();
+      LOGGER.info("Application configured: '{}'", appConfig);
+    } catch (AppConfigurationException e) {
+      System.err.println(e.getMessage());
```

Comment:
Looks like we are already doing this in the pipeline but my take is that we stick to a single logging paradigm. It is too easy for debugging information to get lost when some pieces of it go to stderr and others go through the logging framework.